en/dev_log/ulog_file_format.md

```diff
@@ -23,14 +29,12 @@ They all correspond to the types in C:
 |double             |  8            |
 |bool, char         |  1            |
 
-Additionally all can be used as an array, eg. `float[5]`.
-In general all strings (`char[length]`) do not contain a `'\0'` at the end.
+Additionally the types can be used as an array : eg. `float[5]`. In case of logging of strings : (`char[length]`), it usually does not contain the termination NULL character `'\0'` at the end.
```

Comment:
Note again, why have all that space around the colon? `an array : eg.`.
The first sentence is an improvement, the second is not. I will fix.